### DISCUSSION TITLE
Ask for help

### DISCUSSION THREAD

**Discussion by ametoyuki098** [userdocs/qbittorrent-nox-static#146] [Q&A] [answered]
Use proot-distro archlinux in termux

After starting the program for a period of time, the log message of "monitoring ip failure" will appear in about two hours, as shown in the figure. The strange thing is that I installed the startup in the same way on another Android device, but I didn't have this problem. Can you tell me how to solve this problem?
![image](https://github.com/userdocs/qbittorrent-nox-static/assets/109204477/27d65bec-8ad9-4123-aa86-7bb4e5fba440)

**Comment by userdocs** [comment, marked as answer]
Honestly, I have no idea.

I don't do anything Android/Termux related.

Wild guess, maybe a power saving thing?

**Reply by ametoyuki098** [reply]
All right. Thank you. I'll try to find the problem again myself.
